cmake/FindThreads.cmake

```diff
@@ -0,0 +1,250 @@
+# - This module determines the thread library of the system.
```

Comment:
Also on Windows? 'cause that will be our main target platform for this producer.